src/Symfony/Component/Cache/Adapter/MemcachedAdapter.php

```diff
@@ -75,4 +83,13 @@ protected function doClear($namespace)
     {
         return $this->client->flush();
     }
+
+    public function __destruct()
+    {
+        if (!$this->client->isPersistent() && method_exists($this->client, 'flushBuffers')) {
+            $this->client->flushBuffers();
```

Comment:
TBH, this method isn't even documented yet, found it when looking through the extension source to determine available options for the documentation PR. It looks like this is supposed to be called if the new `buffer_writes` option is enabled.